release-team/role-handbooks/ci-signal/README.md

```diff
@@ -58,29 +58,30 @@ Additionally, the following qualifications make a candidate more suitable for th
 In addition to the above requirements for Shadows, most of which become prerequisites, CI Signal Leads must:
 
 - Have the ability to add a milestone to issues, so must be a member of the [milestone maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers)
-- Have a working knowledge of our various test infrastructure tools, such as Testgrid, Triage, gubernator, Prow, and Tide.
+- Have a working knowledge of our various test infrastructure tools, such as Testgrid, Triage, Spyglass, gubernator, Prow, and Tide.
 - Signal lead need to understand what tests matter and generally how our testing infra is wired together.
   - He/she can ask previous CI Signal leads for advice
   - He/she can ask SIG-Testing for guidance
 
 ## Overview of tasks across release timeline
 
-For any release, its schedule and activities/deliverable for each week will be published in the release directory, e.g: [1.11 schedule](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.11/release-1.11.md#timeline). This section talks about specific CI Signal lead deliverable for each milestone in the release cycle.
+For any release, its schedule and activities/deliverable for each week will be published in the release directory, e.g: [1.11 schedule](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.16#timeline). This section talks about specific CI Signal lead deliverable for each milestone in the release cycle.
```

Comment:
ooops, need to update the text to 1.16 as well

Reply:
Fixed